### PR TITLE
fix: Minting NFT - "dancing"

### LINKED ISSUE #11069: Minting NFT - "dancing"
Look at the text Minting NFT. What is this weird dance? Jumping back and front? Is our minting NFT autistic? 

https://github.com/user-attachments/assets/de365911-d0fc-45ce-a6e7-19fb5e34c7fb

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

if you open the signing modal the **second time**, that section is not fixed starts "dancing"

- [x] Closes #11069

## How to reproduce 

````
<template>
  <SigningModal
    :title="'asd'"
    :is-loading="isLoading"
    :status="status"
  />

  <NeoButton @click="open">
    open
  </NeoButton>
</template>

<script lang="ts" setup>
import { NeoButton } from '@kodadot1/brick'
import type { TransactionStatus } from '@/composables/useTransactionStatus'

definePageMeta({
  layout: 'no-footer',
})

const isLoading = ref(false)
const status = ref(TransactionStatus.Unknown)
const opened = ref(false)

const sleep = (ms: number) => {
  return new Promise(resolve => setTimeout(resolve, ms))
}

const open = async () => {
  isLoading.value = true

  status.value = TransactionStatus.Casting
  await sleep(400)

  status.value = TransactionStatus.Broadcast
  await sleep(1000)

  status.value = TransactionStatus.Block
  await sleep(2000)

  if (opened.value) {
    return
  }

  status.value = TransactionStatus.Finalized
  opened.value = true
}
</script>

````

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

### ❌

https://github.com/user-attachments/assets/8261885e-27b6-4e8b-9436-e612f6f9f28b

### ✅

https://github.com/user-attachments/assets/adde6dca-6c1f-4654-ba08-0c577bf5d1da

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

File: components/shared/SkeletonLoader.vue [hunk 1]
```diff
@@ -82,15 +82,16 @@ const calculateTextContainerWidth = () => {
     return
   }
 
+  textContainerWidth.value = undefined
+
   nextTick(() => {
     const title = titleRef.value?.clientWidth || 0
     const substitle = subtitleRef.value?.clientWidth || 0
```
Comment (vikiival) [inline components/shared/SkeletonLoader.vue:89]:
```suggestion
    const subtitle = subtitleRef.value?.clientWidth || 0
```

File: components/shared/SkeletonLoader.vue [hunk 2]
```diff
@@ -82,15 +82,16 @@ const calculateTextContainerWidth = () => {
...
     const subtitlePlusDots = substitle + DOTS_PLUS_MARGIN_WIDTH
 
-    textContainerWidth.value
-      = substitle && subtitlePlusDots > title
-        ? `${subtitlePlusDots}px`
-        : undefined
+    if (substitle && subtitlePlusDots > title) {
```
Comment (vikiival) [inline components/shared/SkeletonLoader.vue:92]:
```suggestion
    if (subtitle && subtitlePlusDots > title) {
```

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
cc @hassnian there is typo in the `subtitle` word

**Comment by hassnian** [comment]
> cc @hassnian there is typo in the `subtitle` word

fixed 0dfd77e

**Pull request merged**